### PR TITLE
fix: add recording column in video tables

### LINKED ISSUE #7893: Video: Add recording column into video tables
Screenshot from 2021-10-06 19-02-31

* Move Blue, Green and Red video room button directly below room name and try to reduce the width of the first column a bit
* Reduce the width of the column "Additional Info" a bit
* Create an additional column "Recordings"
* In the column "Recordings" show the following
   * Number of Participants
   * Start Time
   * End Time
   * Duration
   * View
* If there are several recordings show the information of several recordings

Note: The information about the video recordings is also displayed in the video edit page on the bottom
Screenshot from 2021-10-06 19-01-03

### PR DESCRIPTION
**Pull request by progmatic-99**
Fixes #7893 

ss_1634129082

### CODE REVIEW

File: app/controllers/events/view/videoroom/list.js [hunk 1]
```diff
@@ -22,52 +25,78 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
...
       },
       {
-        name          : this.l10n.t('Video Source URL'),
-        valuePath     : 'videoStream',
-        helperInfo    : this.l10n.t('This column shows the original link of the video. We do not recommend to share this link as users can access it without logging into') + this.settings.appName + '.com.',
+        name       : this.l10n.t('Video Source URL'),
+        valuePath  : 'videoStream',
+        helperInfo :
+          this.l10n.t(
+            'This column shows the original link of the video. We do not recommend to share this link as users can access it without logging into'
```
Comment (maze-runnar) [inline app/controllers/events/view/videoroom/list.js:53]: why these kind of changes?

File: app/controllers/events/view/videoroom/list.js [hunk 2]
```diff
@@ -8,7 +8,10 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
 
   @action
   async toggleVideoRoom() {
-    this.set('model.event.isVideoroomEnabled', !this.model.event.isVideoroomEnabled);
+    this.set(
+      'model.event.isVideoroomEnabled',
+      !this.model.event.isVideoroomEnabled
+    );
     try {
```
Comment (maze-runnar) [inline app/controllers/events/view/videoroom/list.js:15]: Don't do these kind of formatting changes

File: app/controllers/events/view/videoroom/list.js (hunk 1 above)
Comment (progmatic-99) [reply app/controllers/events/view/videoroom/list.js:53]: Pre-commit hook

Comment (progmatic-99) [reply app/controllers/events/view/videoroom/list.js:53]: Also, the hook follows the standard ES7 rules for writing code.

**Review by maze-runnar — APPROVED** [review]

**Comment by mariobehling** [comment]
Looks like this PR caused an issue here https://github.com/fossasia/open-event-frontend/issues/8066

**Pull request merged**